app/models/builders/manual_builder.rb

```diff
@@ -1,10 +1,6 @@
 require "securerandom"
 
 class ManualBuilder
-  def self.create
-    ManualBuilder.new
-  end
-
```

Comment:
Minor: In the commit message, I would say you are "in-lining" the method, not just removing it.